var/spack/repos/builtin/packages/conquest/package.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+class Conquest(MakefilePackage):
+    """CONQUEST is a DFT code designed for large-scale calculations, 
+    with excellent parallelisation."""
+
+    homepage = "http://www.order-n.org/"
+    url = "https://github.com/OrderN/CONQUEST-release/archive/refs/tags/v1.2.tar.gz"
+    git = "https://github.com/OrderN/CONQUEST-release/"
+
+    # notify when the package is updated.
+    maintainers("davidbowler", "tkoskela", "ilectra")
+
+    version("1.2", sha256="74d974f20ec15ff31d97cd42aae6dbe95288eedfa785896d5872b9ff44ee7ae2")
+    version("1.1", sha256="772e058f073cccfee45521aa62bb13192ab07cb2979b6076ddbf21ba22f9ba5d")
```

Comment:
Would it be possible to manually upload tarballs for these releases? That would ensure more stability of the sha256, see https://github.com/orgs/community/discussions/46034?sort=top for more details.

Reply:
Apologies, I have probably been too brief :slightly_smiling_face: I noticed the maintainers of the Spack package are also the developers of the package. Currently the package relies on autogenerated tarballs from GitHub, but those don't grant stability of the shasum (se discussion linked above). 

I was wondering if it is fine with you to add an explicit artifact to each conquest release, and point the url in this package recipe to that newly uploaded artifact. Uploading should be possible when you edit the release in the Conquest repository. 

If, for any reason, you don't like the suggestion, let me know and I'll merge the package as is

Reply:
Thanks for the explanation! That shouldn't be a problem, but let me check with the main developer

Reply:
Should I keep the sha256 fields in the versions, or remove them?

Reply:
You should keep them, and update the url and shasum.